### ISSUE TITLE
Github - Error trying to contact Github. Please try again

### ISSUE THREAD

**Issue by ward0** [pymedusa/Medusa#10140]
**Medusa (please complete the following information):**
<html>
<body>

Medusa Info: | Branch: masterCommit: f35a19da6bc4efcafff8facca6a4d5b9e3f3f501Version: 0.5.20Database: 44.18
-- | --
Python Version: | 3.8.12 (default, Oct 22 2021, 21:05:38) [GCC 7.5.0]
SSL Version: | OpenSSL 1.1.1l 24 Aug 2021
OS: | Linux-4.4.180+-x86_64-with-glibc2.2.5
Locale: | en_US.UTF-8
Timezone: | CET

</body>
</html>

**Debug logs (at least 50 lines):**
General > Advanced Settings > Enable debug
<details>

```
2021-11-30 16:25:21 WARNING	Thread :: [f35a19d] Error trying to contact Github. Please try again
2021-11-30 16:25:21 DEBUG	Thread :: [f35a19d] Issue submitter failed with error: GithubException(401, {'message': 'Requires authentication', 'documentation_url': 'https://docs.github.com/rest/reference/gists#create-a-gist'})
```
</details>

**Comment by BenjV** [comment]
Most likely a hikup in the network connection to github.
The fact that you can post this issue proves that github is working end you can reach it.

**Comment by ward0** [comment]
@BenjV I have this error 24/7 in Medusa, and yes i can reach github w/o problems from my browser.
I am 200% sure that my username and password is correct.

**Comment by medariox** [comment]
Username and password auth doesn't work anymore (deprecated by GH). You need to use the Personal access token.

**Comment by p0psicles** [comment]
Yeah I'll remove the username/password option from the UI. And add a config migration that will reset user/pass auth to none gh auth set

**Comment by ward0** [comment]
@p0psicles Thanks! I will change to access token after your commit.

**Comment by p0psicles** [comment]
You can already change to token auth. It already works.

**Comment by ward0** [comment]
@p0psicles I just tried with token, clicking on 'generate token' opens a new tab with url: https://dereferer.org/?https%3A%2F%2Fgithub_com%2Fsettings%2Ftokens%2Fnew%3Fdescription=Medusa&scopes=gist%2Cpublic_repo but this does nothing.

**Comment by ward0** [comment]
@p0psicles Yes, i have 2 tabs open, Github and Medusa.
I get this tab now:
![image](https://user-images.githubusercontent.com/15545810/144988792-359ed587-0ddd-44b2-b621-e9124c807b1d.png)

How should i proceed? Do i need to paste this whole page in github somewhere?

**Comment by BenjV** [comment]
I would suggest, just don't use that feature (leaf fields blank) and submit problems (if you experiences one) by hand.

**Comment by cheese1** [comment]
is dereferer.org still operational (as dereferer?)

**Comment by OmgImAlexis** [comment]
> is dereferer.org still operational (as dereferer?)

https://github.com/pymedusa/Medusa/issues/10201
